File: Codingbat/Java/Logic-2/noTeenSum.java
public int fixTeen(int n){
  if(n == 15 || n == 16) return n; 
  else return 0;
}

public int noTeenSum(int a, int b, int c) {
  int total = 0; 
  int[] nums = new int[] {a, b, c}; 
  for( int num : nums){
    if( num >= 13 && num <= 19){
      total += fixTeen(num); 
    } else {
      total += num;
    }
  }
  return total;
}

File: Codingbat/Python/Logic-2/no_teen_sum.py
def no_teen_sum(a, b, c):
  nums = [a,b,c]
  return sum([fix_teen(num) for num in nums])
  
def fix_teen(n):
  if n == 15 or n == 16: 
    return n
  if n <= 12 or n >= 20: 
    return n 
  return 0
